configure.php

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env php
+<?php
+
+function ask(string $question, string $default = ''): string
+{
+    $answer = readline($question . ($default ? " ({$default})" : null) . ': ');
+
+    if (! $answer) {
+        return $default;
+    }
+
+    return $answer;
+}
+
+function confirm(string $question, bool $default = false): bool
+{
+    $answer = ask($question .' (' . ($default ? 'Y/n' : 'y/N') . ')');
+
+    if (! $answer) {
+        return $default;
+    }
+
+    return strtolower($answer) === 'y';
+}
+
+function writeln(string $line): void
+{
+    echo $line.PHP_EOL;
+}
+
+function slugify(string $subject): string
+{
+    return strtolower(trim(preg_replace('/[^A-Za-z0-9-]+/', '-', $subject), '-'));
+}
+
+function titleCase(string $subject): string
+{
+    return str_replace(' ', '', ucwords(str_replace(['-', '_'], ' ', $subject)));
+}
+
+function replaceInFile(string $file, array $replacements): void
+{
+    $contents = file_get_contents($file);
+
+    file_put_contents($file, str_replace(
+        array_keys($replacements),
+        array_values($replacements),
+        $contents
+    ));
+}
+
+function separator(string $path): string
+{
+    return str_replace('/', DIRECTORY_SEPARATOR, $path);
+}
+
+function run(string $command): string
+{
+    return trim((string) shell_exec($command));
+}
+
+function getReplaceableFiles(): array
+{
+    $namedFiles = explode(PHP_EOL, run('find ./* -name "pressbooks-plugin-scaffold*"'));
+
+    $contentFiles = explode(PHP_EOL, run('grep -E -r -l -i "PressbooksPluginScaffold|pressbooks-plugin-scaffold|Pressbooks Plugin Scaffold" --exclude-dir=vendor ./* ./.github/* | grep -v '.basename(__FILE__)));
+
+    return array_unique(
+        array_merge($namedFiles, $contentFiles)
+    );
+}
+
+// --------------- SCRIPT START --------------- //
+$currentDirectory = getcwd();
+
+$pluginName = ask('Plugin name', basename($currentDirectory));
```

Comment:
This is a bit counterintuitive as what's being asked for here is the display name of the plugin (e.g. "Pressbooks Multi-Institutions") but the default shown here is `pressbooks-plugin-scaffold`, the directory name. I entered my plugin directory name (`pressbooks-test-plugin`) and then that string showed up as the top level heading in the README and in the plugin main file header. I'd just hardcode `Pressbooks Plugin Scaffold` as the default so there's a clearer example of what's being asked for.

Reply:
I opted to add this behaviour because one could run something like this `composer create-project pressbooks/pressbooks-plugin-scaffold my-awesome-plugin`. So in this case we should suggest the name based on the directory itself. Maybe we could adjust what's shown to be `Pressbooks Test Plugin` instead of the directory name.

Reply:
I like your solution to this!